bool get_actionmode_req
---
xbot_talker/actionmode_ros actionmode_list